sky/skylet/providers/oci/config.py

```diff
@@ -42,6 +41,14 @@ class oci_conf:
     MAX_RETRY_COUNT = 3
     RETRY_INTERVAL_BASE_SECONDS = 5
 
+    # disk_tier to OCI VPU mapping
+    BOOT_VOLUME_VPU = {
+        None: 10,  # Default, Balanced performance
+        "low": 10,  # 60 IOPS/GB, Balanced performance
+        "medium": 20,  # 75 IOPS/GB, Higher performance
+        "high": 50,  # 120 IOPS/GB, Ultra-high performance
```

Comment:
Thanks for annotating the performance here. How does these number translate to the performance written in the table https://github.com/skypilot-org/skypilot/pull/1812?

Can we have a rough idea of whether these choices match the numbers in https://github.com/skypilot-org/skypilot/blob/1fc6216692872e0d9e0f5d8d74a839a5becc13e2/docs/source/reference/yaml-spec.rst#L77-L80

Reply:
Actually I have a question on these figures: they should be calculated based on a specific disk size, right? Then what's the disk size It used for those figures? @Michaelvll

Reply:
I assume those numbers are retrieved with the default disk_size: 250 GB. cc'ing @cblmemo

Reply:
Yes. It is benchmarked in default disk size, 256GB